guide-projects/spring-boot-eventeria-guide/src/main/java/com/navercorp/eventeria/guide/boot/domain/PostCreatedEvent.java

```diff
@@ -0,0 +1,32 @@
+package com.navercorp.eventeria.guide.boot.domain;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import com.navercorp.eventeria.messaging.contract.event.AbstractDomainEvent;
+
+@Getter
+@ToString(callSuper = true)
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class PostCreatedEvent extends AbstractDomainEvent {
```

Comment:
It would be good to add examples of partition settings for each message.

Reply:
I'll add it in next PR